### DISCUSSION TITLE
How could I use bit to build a micro-fronted.  use vue3 to build host-app, and vue2 component as a part of it

### DISCUSSION THREAD

**Discussion by jiliangmason** [teambit/bit#9005] [FAQ]
how could i use bit to build a micro-fronted.  use vue3 to build host-app, and vue2 component as a part of it? Should I use an additional tool to compile vue2 to vue3?

**Comment by itaymendel** [comment]
hi, this highly depends on what is your approach to microfrontends and the tools you want to use.
for example, you can use module-federation as shown here - https://bit.cloud/frontend/module-federation
but this is just one option out of many. and the final call on the technique to implement it depends on your architecture decisions.
